docs/contribution_workflow.md

```diff
@@ -76,6 +76,10 @@ Each pull request to `master` must pass the following checks.
 Pull requests into a fork will not trigger all of these checks, but you can manually request them 
 individually from the pipeline definition page.
 
+In order to have your change automatically merge once all checks have passed (including optional 
```

Comment:
Another thing probably worth mentioning is we have an eight hour (or something similar) cool down time before it actually merging in the changes.

Reply:
Added a note about that and a link to the Fabric Bot information about that.